<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inviteds</class>
 <widget class="QDialog" name="Inviteds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>5</y>
     <width>481</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Convidados </string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>331</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>18</number>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>238</x>
     <y>374</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonRemover">
      <property name="text">
       <string>Remover</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonEditar">
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonNovo">
      <property name="text">
       <string>Novo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>pushButtonRemover</tabstop>
  <tabstop>pushButtonEditar</tabstop>
  <tabstop>pushButtonNovo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
